toponetx/datasets/graph.py

```diff
@@ -0,0 +1,28 @@
+from torch_geometric.utils.convert import to_networkx
```

Comment:
We should not depend on torch_geometric, because users of toponetx do not want to download torch. instead, we can use another data source for the karate club graph.